libraries/Microsoft.Bot.Builder.AI.QnA/DialogHelper.cs

```diff
@@ -0,0 +1,188 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    /// <summary>
+    /// QnAMaker Active Learning Dialog helper class
+    /// </summary>
+    public class DialogHelper
+    {
+        /// <summary>
+        /// QnA Maker Active Learning dialog name
+        /// </summary>
+        public const string ActiveLearningDialogName = "active-learning-dialog";
+
+        /// <summary>
+        /// Gets QnA Maker Active Learning Dialog.
+        /// </summary>
+        public WaterfallDialog QnAMakerActiveLearningDialog { get; }
```

Comment:
Could you explain why you expose this? What operations do external people do with this?

Exposing complex objects tends to lead to violations of the [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter), which are many times justified, but other times simply increase the complexity of the code. I'm not saying this is bad at this point, but want to understand why we want to do this.

Reply:
We needed this variable to public to initialize the dialog.

Reply:
I'd like to discuss this in person today, marking unresolved

Reply:
We resolve this by creating the QnaMakerBuilder class